tools/windows/install-help/cal/customaction-tests/CustomActionDataTest.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "gtest/gtest.h"
```

Comment:
including something before the precompiled header is a bit worrisome.  Perhaps in the right place in the PCH?

Reply:
I would just swap the two includes.